staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -197,6 +198,8 @@ func NewCmdDrain(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobr
 	cmd.Flags().StringVarP(&o.drainer.PodSelector, "pod-selector", "", o.drainer.PodSelector, "Label selector to filter pods on the node")
 	cmd.Flags().BoolVar(&o.drainer.DisableEviction, "disable-eviction", o.drainer.DisableEviction, "Force drain to use delete, even if eviction is supported. This will bypass checking PodDisruptionBudgets, use with caution.")
 	cmd.Flags().IntVar(&o.drainer.SkipWaitForDeleteTimeoutSeconds, "skip-wait-for-delete-timeout", o.drainer.SkipWaitForDeleteTimeoutSeconds, "If pod DeletionTimestamp older than N seconds, skip waiting for the pod.  Seconds must be greater than 0 to skip.")
+	cmd.Flags().Int64Var(&o.drainer.ChunkSize, "chunk-size", o.drainer.ChunkSize,
```

Comment:
Let's use a helper here (hopping to clean up all these duplicate strings one day...).

https://github.com/kubernetes/kubernetes/blob/60387551078a93baddea8b1012ff0adedc10cae2/staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go#L437